### PR TITLE
Fix Github Link in Side Navigation Menu

### LINKED ISSUE #17: Github Link in Side Navigation Menu is Broken
As the title says. I'm uploading a PR that'll solve this real soon.

### PR DESCRIPTION
**Pull request by zysim**
Closes: #17

### CODE REVIEW

**Comment by zysim** [comment]
If anyone would like to review this feel free to, otherwise I'll go ahead and merge this. It shouldn't break anything.

File: _includes/navigation-sliding.html [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-    {% for link in site.data.navigation %}<li>
-      <a href="{{ site.url }}{{ link.url }}">
-        {% if link.image %}<img src="{{ site.url }}/images/{{ link.image }}" alt="teaser" class="teaser">{% endif %}
-        <div class="title">{{ link.title }}</div>
-        {% if link.excerpt %}<p class="excerpt">{{ link.excerpt }}</p>{% endif %}
-      </a>
-    </li>{% endfor %}
+    {% for link in site.data.navigation %}
+      <li>
+        {% if link.url contains 'http' %}
```
Comment (takluyver) [inline _includes/navigation-sliding.html]: Is there a `startswith` operation or similar? Just to be thorough...

Comment (zysim) [reply _includes/navigation-sliding.html]: Let me check

Comment (zysim) [reply _includes/navigation-sliding.html]: No, there are no such operators. Going through the Liquid docs, it doesn't seem I can make a custom one either.

Comment (zysim) [reply _includes/navigation-sliding.html]: I could write a custom filter or tag though.

Comment (zysim) [reply _includes/navigation-sliding.html]: https://github.com/Maritime-Robotics-Student-Society/blog/pull/18/commits/0a2988acfb269ab21d08f4d42ff4db7c498221c4
^ This should solve it. I've tested it on my machine and it works on my end.

I'll work on a generator for creating the anchors later.

Comment (zysim) [reply _includes/navigation-sliding.html]: https://github.com/Maritime-Robotics-Student-Society/blog/pull/18/commits/2cd35c2072aa46f2129170583739e9d2945d2735 also handles the Github link, where we want the browser to open a new tab when we click on it. Admittedly not the most elegant solution.

**Pull request merged**